docs/sources/mimir/configure/configure-experimental-promql-functions.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Configure experimental PromQL functions
+description: Control what experimental PromQL functions are enabled for your Mimir installation.
+weight: 110
+---
+
+# Configure experimental PromQL functions
+
+Experimental PromQL functions are disabled by default in Mimir. They can be selectively enabled per-tenant or globally for all users.
```

Comment:
[nit] Could we list or link to a list of experimental PromQL functions?

Might also be useful to include the error message that is returned when an experimental PromQL function is used so that someone searching for it sees this page.

Reply:
> [nit] Could we list or link to a list of experimental PromQL functions?

Yep, but there's the danger of it going out of date. Not sure how much of an issue that is in practice. 

> Might also be useful to include the error message that is returned when an experimental PromQL function is used so that someone searching for it sees this page.

Good idea!